docs/src/layouts/AppHeader.tsx

```diff
@@ -26,11 +26,11 @@ const Header = styled('header')(({ theme }) => [
     zIndex: theme.zIndex.appBar,
     backdropFilter: 'blur(20px)',
```

Comment:
```suggestion
    backdropFilter: 'blur(8px)',
```

It might as well help a bit reducing the amount of blur.